e2e/tests/components/summary-list/default.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { expect, test } from '../../../fixtures'
+
+test.describe('Summary list, default', () => {
```

Comment:
```suggestion
test.describe('summary list, default', () => {
```

Sorry, we should probably make the unit test ones lower case as well so they're consistent.